src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -1333,11 +1318,13 @@ namespace IceThermalStorage {
             PlantUtilities::ScanPlantLoopsForObject(state,
                                                     this->Name,
                                                     DataPlant::PlantEquipmentType::TS_IceDetailed,
-                                                    this->PlantLoopNum,
-                                                    this->PlantLoopSideNum,
-                                                    this->PlantBranchNum,
-                                                    this->PlantCompNum,
-                                                    errFlag);
+                                                    this->plantLoc,
+                                                    errFlag,
+                                                    Optional<const Real64>(),
+                                                    Optional<const Real64>(),
+                                                    ObjexxFCL::Optional_int(),
+                                                    ObjexxFCL::Optional_int_const(),
+                                                    ObjexxFCL::Optional_int_const());
```

Comment:
I am getting a "too few arguments" error when I delete these "Optional" stuff. I thought this would work without them, like how it is in the current develop.